auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue

```diff
@@ -729,6 +756,11 @@ export default defineComponent({
       return allowableBusinessPasscodeTypes.includes(props.businessLegalType) && hasBusinessAuthentication.value
     })
 
+    const noAuthenticationOptions = computed(() => {
+      return !((hasAffiliatedAccount.value && enableDelegationFeature.value) ||
+        showEmailOption.value || isBusinessLegalTypeFirm.value || showPasscodeOption.value)
```

Comment:
Just confirming here, it looks like if any option email or passcode is present it would trigger no authentication options? Is that correct?

Reply:
yes

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/d894d8df-450d-4400-96e6-7cd7f8aa67a8)

Reply:
my fix:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/661a12a0-6568-469c-bfbd-0658938cbd3e)